packages/connector-types/src/index.ts

```diff
@@ -2,6 +2,22 @@ import { Language } from '@logto/phrases';
 import { Nullable } from '@silverhand/essentials';
 import { z } from 'zod';
 
+/**
+ * Connector is auto-generated in @logto/schemas according to sql file.
+ * As @logto/schemas depends on this repo (@logto/schemas), we manually define Connector type again as a temporary solution.
```

Comment:
instinctually i think somehow we can remove the dependency from schemas -> types. let's add an issue for this